js/src/dom/event-handler.js

```diff
@@ -22,6 +22,7 @@ const customEvents = {
   mouseenter: 'mouseover',
   mouseleave: 'mouseout'
 }
+const customEventsRegex = /^(mouseenter|mouseleave)/i
```

Comment:
childish comment: better leave it as string inline, instead of use a var. Is very specific (either way is ok)

Reply:
It doesn't matter, though, since the minifier should already inline it :)